server/root.go

```diff
@@ -22,6 +24,24 @@ import (
 	_ "github.com/maticnetwork/heimdall/server/statik"
 )
 
+func StartRestServer(cdc *codec.Codec, registerRoutesFn func(*lcd.RestServer), restCh chan struct{}) error {
+	rs := lcd.NewRestServer(cdc)
+	registerRoutesFn(rs)
+	logger := tmLog.NewTMLogger(log.NewSyncWriter(os.Stdout)).With("module", "rest-server")
+	go restServerHealthCheck(restCh)
```

Comment:
What happens if the daemon is not running? Does it crashes? I do not think so because I have seen heimdall stopping before and the rest server still running (I think).

Reply:
Rest server doesn't crash without Heimdall daemon, but bridge panics if rest server hasn't been started, that's why we put in the health check.

Reply:
I see now, you use rest as a done channel to notify when rest server is running.